addons/account/models/account_move.py

```diff
@@ -3157,26 +3157,30 @@ def add_file_data_results(file_data, invoice):
                 close_file(file_data)
                 continue
 
-            decoder = self._get_edi_decoder(file_data, new=new)
-            if decoder:
-                try:
-                    with self.env.cr.savepoint():
-                        with current_invoice._get_edi_creation() as invoice:
+            try:
+                with self.env.cr.savepoint() as savepoint:
```

Comment:
Note that we need to limit the number of savepoints as much as possible (because of postresql internal limitations), this is going in the opposite direction.

Reply:
Indeed, it's not a good use of resources, I removed that part